source/sass/components/multipage-nav.scss

```diff
@@ -30,6 +30,9 @@
   &.vertical-nav {
     .multipage-link-wrapper {
       margin-bottom: 24px;
+      .multipage-link:not(:active):not(.active) {
+        color: $gray-60;
```

Comment:
This override affects the hover state styling (see "poor employee ..." link)

<img width="233" alt="image" src="https://user-images.githubusercontent.com/2896608/83569080-d2845300-a4d8-11ea-967f-995c1242f187.png">

If our goal is to update the default `.multipage-link` colour, we can just update line 5 in `nav.scss` to `$gray-60`.